frontend/src/metabase/ui/theme.ts

```diff
@@ -12,12 +12,30 @@ import {
   getTitleOverrides,
 } from "./components";
 
+type ThemeColors = MantineTheme["colors"]["brand"];
+
+const getThemeColors = (colors: string[]): ThemeColors => {
```

Comment:
I'm not sure how the new 10 color shades would work in Mantine. Seems like we're filling undefined color with `transparent` instead as if we don't care about them.

Do you think we'll eventually generate 10 shades per color as Mantine 7 does? Or are we going to keep using predefined shades and ignore the 10 shades from Mantine. 

Also, it has a [color generator](https://v7.mantine.dev/changelog/7-0-0#colors-generator) which generates 10 shades of color based on a single input color that could be helpful.

Reply:
I think we explicitly don't want to have 10 shades as in Figma there are only few; other shades shouldn't be available at all